src/rt/sections_solaris.d

```diff
@@ -10,7 +10,16 @@
 
 module rt.sections_solaris;
```

Comment:
the module documentation at the top of this file

Reply:
Sure.

Reply:
Done.